articles/databox/data-box-disk-overview.md

```diff
@@ -99,6 +99,8 @@ Currently Data Box Disk can transfer data to the following Azure regions:
 |East US      |Australia Central |
 |East US2     |Australia Central 2|
 |Japan East     |Korea Central |
+|Japan West     |Korea South |
+|Southeast Asia
```

Comment:
@madhurinms Southeast Asia needs to have an ending |. Hence, this is not rendered correctly. Please fix this and I can sign off on the merge. Thanks Madhuri!